chingu-coders/Voyage2-Turtles-02
# new logo, re-design stickynote titles

## Purpose
* Give DevTab a branding logo
* Make stickynotes more user-friendly

## Approach/Fixes
* Created new logo
* Re-designed sticky note title to improve UI/UX

## Testing
* Pull code to your local machine
* Check to see that new logo replaces the turtle logo
* Open a new sticky note and see the title for instructions
* Test: Double click the top bar section to change the title
* Test: new title submits on `enter` as well as clicking outside of the input field
* Test: Highlight over the title to notice a small `x` in the top right corner (this deletes the title)
